<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KinectViewer</class>
 <widget class="QMainWindow" name="KinectViewer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>601</width>
    <height>407</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>5000</width>
    <height>5000</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>KinectViewer</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QCheckBox" name="checkBox_savePointCloud">
      <property name="text">
       <string>Save PointClouds</string>
      </property>
     </widget>
    </item>
    <item row="7" column="0">
     <widget class="QPushButton" name="btnResetCamera">
      <property name="text">
       <string>Zoom to fit</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QCheckBox" name="checkBox_includeOrientation">
      <property name="text">
       <string>Include Orientation</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QCheckBox" name="checkBox_startStream">
      <property name="text">
       <string>Show Stream</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="3" column="0">
     <widget class="QPushButton" name="pushButton_save">
      <property name="text">
       <string>SaveAllPC</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2" rowspan="9">
     <widget class="QVTKWidget" name="qvtkWidget" native="true"/>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QVTKWidget</class>
   <extends>QWidget</extends>
   <header>QVTKWidget.h</header>
   <container>1</container>
   <slots>
    <slot>setEnable(int)</slot>
   </slots>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
